packages/nuxt/test/vite/sourceMaps.test.ts

```diff
@@ -148,146 +151,187 @@ describe('getPluginOptions', () => {
       }),
     );
   });
+
+  it('sets filesToDeleteAfterUpload correctly based on fallback options', () => {
```

Comment:
super-l: is there a reason to test multiple scenarios in one test? no strong feelings but I personally prefer sticking to one scenario per test (could be `it.each` or separate tests)  because it makes it easier to see what specifically fails. But feel free to disregard and stick with this.